packages/api-client/src/api/addAddress/index.ts

```diff
@@ -3,9 +3,9 @@ import getCurrentBearerToken from '../authentication/getCurrentBearerToken';
 import { serializeAddress } from '../serializers/address';
 
 export default async function addAddress({ client, config }: ApiContext, params: Address): Promise<void> {
-  const bearerToken = await getCurrentBearerToken({ client, config });
+  const { bearer_token } = await getCurrentBearerToken({ client, config });
```

Comment:
This causes a linter error because `bearer_token` is not in camel case. However, this problem will disappear once we rewrite the sdk functions to use new interfaces. Then it will look something like this:
```ts
const token = await getCurrentBearerToken({ client, config });
const result = await client.account.createAddress({ ...token, address: serializedAddress });
```